src/components/pages/donations.tsx

```diff
@@ -154,136 +164,39 @@ export function Donations() {
           </p>
           <p>
             Du kannst auch via{' '}
-            <StyledA href="#no-formular">Überweisung oder PayPal</StyledA>{' '}
+            <a className="serlo-link" href="#no-formular">
+              Überweisung oder PayPal
+            </a>{' '}
             spenden.
           </p>
           <br />
-        </ContactCol>
-      </Section>
-
-      <BlueContainer>
+        </div>
+      </section>
+      <BlueContainer
+        className={clsx(
+          'bg-brand text-white mt-24 p-7 serlo-responsive-padding'
+        )}
+      >
         <PartnerList />
       </BlueContainer>
       <FooterNav data={footerEntries} />
     </>
   )
 }
 
-const BlueHeader = styled.header`
-  ${makePadding}
-  padding-top: 46px;
-  background-color: ${(props) => props.theme.colors.bluewhite};
-
-  @media (min-width: ${(props) => props.theme.breakpoints.sm}) {
-    text-align: center;
-  }
-`
-
-const PaddedDiv = styled.div`
-  padding-bottom: 24px;
-`
-
-const Section = styled.section`
-  margin-top: 20px;
-  margin-bottom: 60px;
-  ${makeResponsivePadding}
-
-  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
-    margin-top: 50px;
-    padding-left: 32px;
-    padding-right: 32px;
-  }
-
+const BlueContainer = styled.div`
   @media (min-width: ${(props) => props.theme.breakpoints.lg}) {
-    display: block;
-  }
-`
-
-const ContactCol = styled.div`
-  @media (max-width: ${(props) => props.theme.breakpointsMax.md}) {
-    border-top: 8px solid ${(props) => props.theme.colors.lightBlueBackground};
-    padding-top: 30px;
-    margin-top: 45px;
-  }
-  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
-    max-width: 300px;
-    position: absolute;
-    right: 0;
-    top: 197px;
-    padding-right: 15px;
+    img {
+      max-height: 60px !important;
+    }
   }
 `
 
 const DonationCol = styled.div`
-  max-width: 600px;
-
-  margin-right: auto;
-  margin-left: auto;
-
-  @media (min-width: ${(props) =>
-      props.theme.breakpoints.md}) AND (max-width: 1320px) {
-    margin-left: inherit;
-  }
-
   iframe {
     margin-left: -20px;
```

Comment:
@elbotho This line is causing the donation form to be clipped:

![grafik](https://user-images.githubusercontent.com/13507950/120674247-3480eb00-c494-11eb-9415-ace6cae4635a.png)

So I have removed it. Any thoughts?

Reply:
Thanks, better this way.